### DISCUSSION TITLE
Wave usage

### DISCUSSION THREAD

**Discussion by rmfjd78** [h2oai/wave#536] [General]
Hi 

I have a question with wave can i make in wave a web page, that is published on a server so users can use it?
If we can do that, are there plots to analyse a dataset regarding correlations and other type of plots.

Best regards,

**Comment by lo5** [comment]
Yes, you can put up your Wave app on a server so that it's accessible by others. 
There are currently no built-in analysis routines in Wave. You'll need to use other Python libraries for that.
